consoles/VNC.pm

```diff
@@ -378,7 +378,7 @@ sub _server_initialization ($self) {
     $self->_pixinfo(\%pixinfo);
     $self->_bpp($supported_depths{$self->depth}->{bpp});
     $self->_true_colour($supported_depths{$self->depth}->{true_colour});
-    $self->_do_endian_conversion($self->no_endian_conversion ? 0 : $server_is_big_endian != $client_is_big_endian);
+    $self->_do_endian_conversion(($server_is_big_endian && $client_is_big_endian) ? 1 : 0);
```

Comment:
When you remove the use of "no_endian_conversion" I recommend you also remove that attribute.

Please also extend the explanation in the git commit message a bit to explain the problem with the former approach and how the current approach is better. Right now it kinda says something like "true is true" which doesn't help me that much ;)

Reply:
@okurz Thanks for the review! I reverted the change with the `no_endian_conversion` attribute and updated the git message. I hope it's more understandable now.